awx/main/migrations/0064_v350_analytics_state.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-01-28 14:27
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0063_v350_org_host_limits'),
+    ]
+
+    operations = [
```

Comment:
Be aware, you're probably going to be racing @AlanCoding to claim this one, I expect he'll beat you in https://github.com/ansible/awx/pull/3266